Sources/PackageModel/Manifest/TargetDescription.swift

```diff
@@ -129,13 +139,15 @@ public struct TargetDescription: Equatable, Codable {
             if url != nil { throw Error.disallowedPropertyInTarget(targetName: name, propertyName: "url") }
             if pkgConfig != nil { throw Error.disallowedPropertyInTarget(targetName: name, propertyName: "pkgConfig") }
             if providers != nil { throw Error.disallowedPropertyInTarget(targetName: name, propertyName: "providers") }
+            if extensionCapability != nil { throw Error.disallowedPropertyInTarget(targetName: name, propertyName: "extensionCapability") }
```

Comment:
similar comment to the precondition one above

Reply:
Should I convert all this code then?  Consistency seemed best here.